spamclassifier/model_trainer.py

```diff
@@ -0,0 +1,46 @@
+from os.path import dirname as dn, join as jp
```

Comment:
Ok, I know, it's really not good names, but at least it shouldn't override existing imports and should avoid PEP8 E501...